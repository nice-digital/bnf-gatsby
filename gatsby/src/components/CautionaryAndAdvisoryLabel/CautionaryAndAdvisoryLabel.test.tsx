import React from "react";

describe("CautionaryAndAdvisoryLabel", () => {
	it.todo("Should render a heading containing the label number");
	it.todo("Should render both English and Welsh recommendations");
	it.todo("Should contain an HTML description of the label");
});
