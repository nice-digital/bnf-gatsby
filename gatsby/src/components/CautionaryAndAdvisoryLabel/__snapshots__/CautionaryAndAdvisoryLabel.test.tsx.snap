// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CautionaryAndAdvisoryLabel should render the correct content within each section 1`] = `
<section
  aria-labelledby="label-3"
>
  <h2
    id="label-3"
  >
    Label 3
  </h2>
  <div
    class="label"
  >
    <p>
      English test recommendation
    </p>
    <p
      class="welsh"
      lang="cy"
    >
      Welsh test recommendation
    </p>
  </div>
  <div>
    <p>
      Test description
    </p>
  </div>
</section>
`;
